examples/pytorch/gin/README.md

```diff
@@ -6,78 +6,28 @@ Graph Isomorphism Network (GIN)
 ```bash
-# 4 bioinformatics datasets setting graph_pooling_type=sum, the nodes have categorical input features 
-python main.py --dataset MUTAG --device 0  \
-                --graph_pooling_type sum --neighbor_pooling_type sum --filename MUTAG.txt
-
-python main.py --dataset PTC --device 0  \
-                --graph_pooling_type sum --neighbor_pooling_type sum --filename PTC.txt
-
-python main.py --dataset NCI1 --device 0  \
-                --graph_pooling_type sum --neighbor_pooling_type sum --filename NCI1.txt
-
-python main.py --dataset PROTEINS --device 0  \
-                --graph_pooling_type sum --neighbor_pooling_type sum --filename PROTEINS.txt
-
-# 5 social network datasets setting graph_pooling_type=mean, for the REDDIT datasets, we set all node feature vectors to be the same 
-# (thus, features here are uninformative); for the other social networks, we use one-hot encodings of node degrees.  
-python main.py --dataset COLLAB --device 0  \
-                --graph_pooling_type mean --neighbor_pooling_type sum --degree_as_nlabel --filename COLLAB.txt
-
-python main.py --dataset IMDBBINARY --device 0  \
-                --graph_pooling_type mean --neighbor_pooling_type sum --degree_as_nlabel --filename IMDBBINARY.txt
-
-python main.py --dataset IMDBMULTI --device 0  \
-                --graph_pooling_type mean --neighbor_pooling_type sum --degree_as_nlabel --filename IMDBMULTI.txt
-
-python main.py --dataset REDDITBINARY --device 0  \
-                --graph_pooling_type mean --neighbor_pooling_type sum --filename REDDITBINARY.txt --fold_idx 6 --epoch 120
-
-python main.py --dataset REDDITMULTI5K --device 0  \
-                --graph_pooling_type mean --neighbor_pooling_type sum --filename REDDITMULTI5K.txt
+python3 train.py --dataset MUTAG
 ```
 
-one fold of 10 result are below.
-
-| dataset       | our result | paper report |
-| ------------- | ---------- | ------------ |
-| MUTAG         | 89.4      | 89.4 ± 5.6   |
-| PTC           | 68.5      | 64.6 ± 7.0   |
-| NCI1          | 78.5      | 82.7 ± 1.7   |
-| PROTEINS      | 72.3      | 76.2 ± 2.8   |
-| COLLAB        | 81.6      | 80.2 ± 1.9   |
-| IMDBBINARY    | 73.0      | 75.1 ± 5.1   |
-| IMDBMULTI     | 54.0      | 52.3 ± 2.8   |
-| REDDITBINARY  | 88.0      | 92.4 ± 2.5   |
-| REDDITMULTI5K | 54.8      | 57.5 ± 1.5   |
+> **_NOTE:_**  Users may observe results fluctuate due to the randomness with relatively small dataset.  In consistence with the original [paper](https://arxiv.org/abs/1810.00826), five social network datasets, 'COLLAB', 'IMDBBINARY' 'IMDBMULTI' 'REDDITBINARY' and 'REDDITMULTI5K', are also available as the input. Users are encouraged to update the script slightly for social network applications, for example, replacing sum readout on bioinformatics datasets to mean readout on social network datasets and using one-hot encodings of node degrees by setting "degree_as_nlabel=True".
```

Comment:
"using one-hot encodings of node degrees by setting "degree_as_nlabel=True"." -> "using one-hot encodings of node degrees by setting "degree_as_nlabel=True" in GINDataset."

Reply:
Okay. I make the update.